projects/ngx-charts-on-fhir/src/lib/fhir-chart/range-selector/range-selector.component.css

```
@@ -1,9 +1,14 @@
 .range-selector {
-    display: flex;
-    align-items: flex-start;
-    gap: 8px;
-  }
-  
-  .range-selector mat-button-toggle-group {
-    margin-top: 2px;
-  }
+  display: flex;
+  align-items: flex-start;
+  gap: 8px;
+}
+
+.range-selector mat-button-toggle-group {
+  margin-top: 2px;
+}
+
+.range-selector mat-form-field {
+  width: 175px;
+   height: 73px;
```

this can be even smaller and there's no need to set the height.
```suggestion
  width: 140px;
```